syntax = "proto3";

package src.shared.network;

import "types.proto";

message ListSyncFoldersResponse {
  repeated src.shared.types.SyncFolder syncFolders = 1;
}

message AddSyncFolderRequest {

}

message RemoveSyncFolderRequest {

}

message StartSyncRequest {
  
}